### ISSUE TITLE
For some users on Firefox toolbox seems to be broken. 

### ISSUE THREAD

**Issue by creesch** [creesch/reddit-moderator-toolbox#788]
They reported settings not functioning and counters not working either. 

Unfortunately @Geo1088 amd I haven't been able to get it to reproduce on several versions of Firefox.

**Comment by agentlame** [comment]
>amd

**Comment by creesch** [comment]
Ok I figured out why so few people seem to have reported issues with firefox. [This set me on the right track](https://redd.it/5eateq), the issue only occurs when using private browsing mode **and** the user is not logged into reddit in regular mode.  In other words, firefox's sandboxing seems to be a bit fucked up. 

To reproduce it: 

- Log out of reddit in regular mode. 
- Open private browsing mode. 
- Log into reddit. 

*Result:* toolbox is not working properly. 

Different scenario: 

- In regular browsing mode log into reddit with account A.
- Open private browsing mode. 
- Log into reddit with account B. 

*Result:* Toolbox seems to work in private browsing mode but is in fact showing counters and other details from account A and **not** from account B. 



tl;dr private browsing firefox is currently fubar.

**Comment by creesch** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1319536

**Comment by creesch** [comment]
Seems like (part of) the issue is fixed in firefox nightly. We will need to do some testing to confirm it. 

I suspect there is more regression going on in firefox 50 as there are a few people having these issues that insist they are not using nightmode.

**Comment by utack** [comment]
Happens for me, not using private mode
Any workaround that works for now?
Thanks

**Comment by creesch** [comment]
The only workaround that sort of seems to work is to completely start from scratch with a clean profile in firefox.

**Comment by creesch** [comment]
The bug on firefox's end is fixed in an upcoming release. From what I understand it should be fixed in firefox 51. 

I can't confirm so far if other issues are fixed.

**Comment by utack** [comment]
Turns out it is not, at least here  
Pretty odd though, on one machine it never broke, on the second machine it is still broken

**Comment by creesch** [comment]
Turns out the issues was fixed in firefox, in most other cases it is due to a firefox profile being corrupted.

**Comment by utack** [comment]
So what can I do when toolbox is still sometimes dead?
Any config files to nuke, to set it up fresh?

**Comment by creesch** [comment]
Start firefox with the -P flag and create a new profile.
